app/presenters/publishing_api_presenters/political_helper.rb

```diff
@@ -0,0 +1,12 @@
+module PublishingApiPresenters::PoliticalHelper
+private
+
+  def government
+    gov = item.government
+    {
+      title: gov.name,
+      slug: gov.slug,
+      current: gov.current?
+    }
```

Comment:
Should `political_details` also merge in the government parts, as they are essentially connected?

```
{
  political: item.political?,
  government: government
}
```